lib/floe/workflow/catcher.rb

```diff
@@ -3,13 +3,13 @@
 module Floe
   class Workflow
     class Catcher
-      attr_reader :error_equals, :next, :result_path
+      attr_reader :error_equals, :next_state, :result_path
 
       def initialize(payload)
         @payload = payload
 
         @error_equals = payload["ErrorEquals"]
-        @next         = payload["Next"]
+        @next_state   = payload["Next"]
```

Comment:
The intention in the naming here, originally, was that the ivars matched the payload names to keep it consistent.

Though it does bring some clarity, I don't thin the name change is needed in this PR, unless I'm missing something.

Reply:
Yeah I think having the name match the ASL property outweighs the keyword collision.  It is only an issue if you want to use the reader method instead of `@next` from within the class.